### DISCUSSION TITLE
Buildpack Semantic Versioning Rules

### DISCUSSION THREAD

**Discussion by genevieve** [paketo-buildpacks/feedback#10] [General]
We maintain our own builder and would benefit from documentation that explains the semantic versioning rules for the paketo buildpacks. Under what circumstances is a buildpacks major, minor, and patch point bumped? This will enable us to version our builder in a corresponding way to capture "breaking" changes, CVE patches, new backwards-compatible features, etc.

**Comment by ForestEckhardt** [comment]
I think this is a great topic for conversation. I am going to put a link to this in for the agenda for next weeks 4/13/2021 to try and get the ball rolling on this sooner rather than later.

**Comment by fg-j** [comment]
This is a *great* question. If we get a good and answer for this, the Paketo builders' semver would be very easy to get right automatically, as well.

**Comment by arjun024** [comment]
I attempted to create an opinionated RFC here in attempt to gather comments/opinions https://github.com/paketo-buildpacks/rfcs/pull/49
